<launch>
    
    <node name="kittiHelper" pkg="aloam_velodyne" type="kittiHelper" output="screen"> 
        <param name="dataset_folder" type="string" value="/media/whu/HD_CHEN_2T/02data/KITTI_odometry/dataset/" />
        <param name="calib_file" type="string" value="/home/whu/slam_ws/src/lv_slam/config/kitti_calib/calib04-12_corrected.txt" /> 
        <param name="sequence_number" type="string" value="04" />
        <param name="to_bag" type="bool" value="false" />
        <param name="output_bag_file" type="string" value="/home/whu/data/aloam_kitti/kitti_0x_aloam.bag" /> <!-- replace with your output folder -->
        <param name="publish_delay" type="int" value="1" />
        <param name="output_path" type="string" value="/home/whu/data/aloam_kitti/output_hz10_calib" />
    </node>
</launch>